test/e2e/apimachinery/aggregator.go

```diff
@@ -90,6 +90,7 @@ var _ = SIGDescribe("Aggregator", func() {
 	})
 
 	/*
+		    Release : v1.17
```

Comment:
nit: Can we get rid of the space between "Release :"?

Reply:
Was following the format that's provided in the [community guide](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-comment-metadata) as shown below.

```
/*
  Release : v1.9
  Testname: Kubelet: log output
  Description: By default the stdout and stderr from the process being
  executed in a pod MUST be sent to the pod's logs.
*/
framework.ConformanceIt("it should print the output to logs", func() {
  ...
})
```

Reply:
ahh got it. It looks like our test for this uses the incorrect format (https://github.com/kubernetes/kubernetes/blob/master/test/conformance/walk_test.go#L153), but that's outside the scope of this PR.

Reply:
I would argue the community guide has a typo in it, I don't think the space should be necessary